### PR TITLE
[FLAG-656] Node version upgrade with Next 11

### PR DESCRIPTION
**Pull request by wri7tno**
#4269

### CODE REVIEW

**Review by willian-viana — APPROVED** [review]
:shipit:

File: components/analysis/component.jsx [hunk 1]
```diff
@@ -80,31 +78,33 @@ class AnalysisComponent extends PureComponent {
...
+                    REFRESH ANALYSIS
+                  </Button>
+                )}
                 <Button
-                  className="refresh-analysis-btn"
-                  onClick={() => handleFetchAnalysis(endpoints)}
+                  className="cancel-analysis-btn"
+                  onClick={handleCancelAnalysis}
                 >
-                  REFRESH ANALYSIS
```
Comment (wri7tno) [inline components/analysis/component.jsx:90]: is this requested in a specific tickret? changing "REFRESH" to "CANCEL", seems it isn't part of the node update ticket 🤔

**Comment by wri7tno** [comment]
LGTM

File: components/analysis/component.jsx (hunk 1 above)
Comment (willian-viana) [reply components/analysis/component.jsx:90]: Nops, it was a trick. If you compare the code nothing changed. It was just the ESLint. I've just checked it this morning to make sure haha

**Pull request merged**